test/unit/xfa_tohtml_spec.js

```diff
@@ -17,18 +17,20 @@ import { isNodeJS } from "../../src/shared/is_node.js";
 import { XFAFactory } from "../../src/core/xfa/factory.js";
 
 describe("XFAFactory", function () {
-  function searchHtmlNode(root, name, value, byAttributes = false) {
+  function searchHtmlNode(root, name, value, byAttributes = false, nth = [0]) {
```

Comment:
*I might be missing something obvious, but this looks confusing to me:*
Does the new parameter *need* be be an Array, or would a simple value suffice (i.e. `nth = 0`) since that'd be a bit more easy to understand?

Reply:
It's just to pass the argument by "reference": the `nth--` won't change the value of `nth` in the caller.